src/lavinmq/client/channel.cr

```diff
@@ -12,10 +12,12 @@ module LavinMQ
       include Stats
       include SortableJSON
 
+      Log = Client::Log.for "channel"
```

Comment:
It's used by the `EntityLog` for `Channel`, and also by `Consumer`.

Following the docs/"pattern" for `Log`, declaring a `Log` per namespace seems to be the way to go. Sticking to always declaring `Log` in module/class makes it consistent on where `source` is set. I have no real opinion on this. Also, as I mention in the other comment, I thought it would create a new `Log` instance for every `for` call.